doc/rtd/topics/debugging.rst

```diff
@@ -163,3 +163,103 @@ commandline:
 
 Inspect cloud-init.log for output of what operations were performed as a
 result.
+
+.. _proposed_sru_testing:
+
+Stable Release Updates (SRU) testing for cloud-init
+===================================================
+Once an Ubuntu release has been published, updates to it must follow a special
+procedure called a "stable release update" (or `SRU`_).
+
+The cloud-init project has a specific process it follows when validating
+a cloud-init SRU, documented in the `CloudinitUpdates`_ wiki page.
+
+Generally an SRU test of cloud-init performs the following:
+
+ * Install a pre-release version of cloud-init from the
+   **-proposed** APT pocket (e.g. **bionic-proposed**)
+ * Upgrade cloud-init and attempt a clean run of cloud-init to assert the new
+   version of cloud-init works properly the specific platform and Ubuntu series
+ * Check for tracebacks or errors in behavior
+
+
+Manual SRU verification procedure
+---------------------------------
+Below are steps that can manually test a pre-release version of cloud-init
+from **-proposed**
+
+.. note::
+    For each SRU, upstream manually validates cloud-init upgrades for **Ubuntu**
+    on these platforms: **Amazon Ec2, Azure, GCE, OpenStack, Oracle,
+    Softlayer (IBM), NoCloud (LXD&KVM)**
```

Comment:
NoCloud isn't really a platform:

```suggestion
    Softlayer (IBM), LXD, KVM**
```

(I mean, LXD and KVM aren't really either, but they're more recognisable.)